src/encoded/viewconfigs/auditview.py

```diff
@@ -131,6 +131,10 @@ def _construct_query(self):
                 ['_all', '*.uuid', '*.md5sum', '*.submitted_file_name']
             )
         audit_field_list, used_filters = self._set_query_aggs(query)
+        if not query['query']:
```

Comment:
I am not following the removal of null parameters, but perhaps I missed a snovault change.

Reply:
This is required by Elasticsearch 6.